modules/environment/aws/eks/iam.tf

```diff
@@ -1,13 +1,3 @@
-resource "aws_iam_openid_connect_provider" "default" {
-  url = module.eks.cluster_oidc_issuer_url
-
-  client_id_list = [
-    "sts.amazonaws.com",
-  ]
-
-  thumbprint_list = ["9e99a48a9960b14926bb7f3b02e22da2b0ab7280"]
-}
-
```

Comment:
removed this because we're setting `enable_irsa` to `true` now in the EKS module